### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Available Software_ Screen Reader: Narrator/NVDA is not reading the “Details” label for the respective text field while shift tabbing.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3359]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #NVDA; #Narrator; #Java tooling for IntelliJ/Eclipse; #C+E-H
 
**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
 
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator, NVDA
**MAS Violated:** MAS1.3.1

 **NOTE:** Same issue is observed in “Install Details” screen as well.

**Repro Steps:** 
1) Start Screen Reader and 0pen Java EE - Eclipse application.
2) Tab till Help-->Install New software.
3)  Enter http://dl.microsoft.com/eclipse/ in 'Work with' text box and press enter.
4)Tab to all controls and verify how screen reader is reading.
5)Verify whether screen reader is reading “Details” label while Shift tabbing.

**Actual Result:** 
Narrator/NVDA is not reading the “Details” label for the respective text field while shit tabbing.

 **Expected Result:** 
Narrator/NVDA should read  the “Details” label for the respective text field while shit tabbing.
 
**User Impact:** 
Screen Reader may get complete information about the filed if screen reader doesn’t read the labels for the fields.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
[Bug_3359_Screen Reader isssue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451493/Bug_3359_Screen.Reader.isssue.pptx)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The install UI is controlled by eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). Here is a gif of install AWS Eclipse Toolkits, we could see the same behavior of this issue

![3359](https://user-images.githubusercontent.com/12445236/64940311-3953d480-d896-11e9-81eb-834d06c54708.gif)

**Comment by Flanker32** [comment]
- Importance of experience: Little, it's not a common behavior to shift back while install eclipse plugins
- Quality of experience: Little, screen reader could read the group for the first time
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us review it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749793
